assets/less/cds-rdm/globals/site.overrides

```diff
@@ -90,3 +90,11 @@
     opacity:1;
   }
 }
+
+.ui.disabled.input, .ui.input:not(.disabled) input[disabled] {
+  opacity: 1 !important;
+}
+
+.ui.form .flashed.message {
+  z-index: 0 !important;
```

Comment:
To avoid error message overlap

Before:
<img width="1275" alt="Screenshot 2024-12-05 at 12 24 47" src="https://github.com/user-attachments/assets/b02ba57c-c621-43c0-a474-c69edac21c3d">

After:
<img width="1327" alt="Screenshot 2024-12-05 at 12 22 13" src="https://github.com/user-attachments/assets/b621d60e-7b40-4a40-be0f-c0a51476703b">